app/src/main/java/com/appculinaryrecipes/MainActivity.java

```diff
@@ -30,7 +47,16 @@ protected void onCreate(Bundle savedInstanceState) {
 //            firebaseAuth.signOut();
 //            checkUser();
 //        });
-        activityMainBinding.navigationBar.setItemSelected(R.id.home,true);
+        activityMainBinding.navigationBar.setItemSelected(R.id.home, true);
+        activityMainBinding.navigationBar.setOnItemSelectedListener(i -> {
```

Comment:
brak przejścia do AddRecipeFragment

Reply:
@klesniewski50 bo nie zostało to zmergowane z twoim chyba jeszcze

Reply:
zostało, mergowałem